chef_master/source/resource_remote_file.rst

```diff
@@ -371,11 +375,26 @@ This resource has the following properties:
 
    Enable If-Modified-Since headers. Set to ``false`` to disable If-Modified-Since headers. To use this setting, ``use_conditional_get`` must also be set to ``true``. Default value: ``true``.
 
-``show_progess`` 
+``show_progess``
    **Ruby Types:** TrueClass, FalseClass
 
    Displays the progress of the file download. Set to ``true`` to enable this feature. Default value: ``false``.
 
+``remote_user``
+   **Ruby Types:** String
+
+   *Windows only:* The user name of a user with access to the remote file specified by the `source` property. Default value: `nil`. The user name may optionally be specifed with a domain, i.e. `domain\user` or `user@my.dns.domain.com` via Universal Principal Name (UPN) format. It can also be specified without a domain simply as `user` if the domain is instead specified using the `remote_domain` attribute. Note that this property is ignored if `source` is not a UNC path. If this property is specified, the `remote_password` property **must** be specified.
+
+``remote_password``
+   **Ruby Types:** String
+
+   *Windows only:* The password of the user specified by the `remote_user` property. Default value: `nil`. This property is mandatory if `remote_user` is specified and may only be specified if `remote_user` is specified. The `sensitive` property for this resource will automatically be set to `true` if `remote_password` is specified.
+
+``remote_domain``
+   **Ruby Types:** String
+
+   *Windows only:* The domain of the user user specified by the `remote_user` property. Default value: `nil`. If not specified, the user and password properties specified by the `remote_user` and `remote_password` properties will be used to authenticate that user against the domain in which the system hosting the UNC path specified via `source` is joined, or if that system is not joined to a domain it will authenticate the user as a local account on that system. An alternative way to specify the domain is to leave this property unspecified and specify the domain as part of the `remote_user` property.
```

Comment:
I'd shorten this:

> If not specified, the user and password properties specified by the `remote_user` and `remote_password` properties will be used to authenticate that user against the domain in which the system hosting the UNC path specified via `source` is joined

to just:
By default the resource will authenticate against the domain of the remote system, or as a local account if the remote system is not joined to a domain.

And:
> An alternative way to specify the domain is to leave this property unspecified and specify the domain as part of the `remote_user` property.

to:

The domain may also be specified as part of the `remote_user` property.